mono/mini/aot-runtime.c

```diff
@@ -3313,9 +3315,11 @@ mono_aot_find_jit_info (MonoDomain *domain, MonoImage *image, gpointer addr)
 
 	nmethods = amodule->info.nmethods;
 
+#ifndef ENABLE_INTERPRETER
```

Comment:
I think the problem is that the interpreter requires some aot-ed wrappers to work right now ?
So it would not work with full-aot+domains.